src/themes/components/modal.ts

```diff
@@ -72,6 +72,37 @@ const baseStyle = definePartsStyle({
   footer: baseStyleFooter,
 });
 
+const variants = {
+  base: baseStyle,
+  fullScreenGallery: {
+    overlay: {
+      bg: 'black',
+      zIndex: 'modal',
+      opacity: `${opacity[40]} !important`,
```

Comment:
Why do we need to override the opacity with important? Also, why would it matter if the modal is full screen isn't the overlay invisible?